src/c++/perf_analyzer/perf_analyzer.h

```diff
@@ -1,4 +1,4 @@
-// Copyright 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+// Copyright 2022-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
is this the only change in this file?

Reply:
now it is. i will revert this file change